network/model.go

```diff
@@ -1,6 +1,9 @@
 package network
 
-type CreatePlayerModel struct {
-	Username    string `json:"username"`
-	DisplayName string `json:"displayName"`
+import "github.com/joshprzybyszewski/cribbage/model"
+
+type CreateInteractionRequest struct {
+	PlayerID model.PlayerID `json:"playerID"`
+	Mode     string         `json:"mode"`
+	Info     interface{}    `json:"info"`
```

Comment:
🌵 🧙 when defining the json tags, we can choose them to be small (rather than human readable), just so long as each field's is unique. That is:

```go
type CreateInteractionRequest struct {
	PlayerID model.PlayerID `json:"pID"`
	Mode     string         `json:"m"`
	Info     interface{}    `json:"i"`
}
```

Reply:
I had another comment about this - making them human readable will make our front end less cryptic when we have to access the fields within JS files. Yes, it means sending less over the wire, but I'm not sure the gain there is worth somewhat obfuscated frontend code?

Reply:
Sounds good. we can leave them human readable